### PR TITLE
Cleanup migrating durable function files

### PR DESCRIPTION
**Pull request by bcarthic**
## Description
Cleanup migrating durable function files

## Related issues
Addresses [[AB#106243](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/106243)].

## Testing
Describe how this change was tested.

### CODE REVIEW

File: src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+SET XACT_ABORT ON
+BEGIN TRANSACTION
+
+DROP PROCEDURE IF EXISTS dbo.GetInstanceBatchesByTimeStamp
+GO
+
+COMMIT TRANSACTION
+
+DROP INDEX IF EXISTS IX_Instance_Watermark_Status_CreatedDate ON dbo.Instance
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql:9]: This was only used for the sproc?

File: src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj [hunk 2]
```diff
@@ -68,6 +68,7 @@
     <TSqlScript Include="Features\Schema\Sql\Scripts\Sequences.sql" />
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
     <TSqlScript Include="Features\Schema\Sql\Types\*.sql" />
+    <None Remove="Features\Schema\Migrations\44.diff.sql" />
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj:71]: Don't think this was intentional 😛

File: src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+GO
+
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql:11]: nit: extra line

File: src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql (hunk 1 above)
Comment (bcarthic) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/44.diff.sql:9]: yeah, It was created as part of this work

**Review by wsugarman — APPROVED** [review]

**Pull request merged**